src/node_contextify.cc

```diff
@@ -1488,11 +1489,14 @@ void ContextifyContext::CompileFunction(
   }
 
   TryCatchScope try_catch(env);
+  std::span<Local<String>> parms(params.data(), params.size());
+  std::span<Local<Object>> exts(context_extensions.data(),
+                                context_extensions.size());
   Local<Object> result = CompileFunctionAndCacheResult(env,
                                                        parsing_context,
                                                        &source,
-                                                       params,
-                                                       context_extensions,
+                                                       parms,
```

Comment:
Is this intentional? Why parms but not params?

Reply:
`params` and `parms` are different types.